finsoc/app/controllers/main_controller.rb

```diff
@@ -12,6 +12,18 @@ def about_us
   end
 
   def newsletter
+    @paths = Dir["app/assets/pdf/newsletters/*"]
```

Comment:
This way is not ideal, but we can fix this when we figure out the Filestack API for uploading files.